cmd/zc_attr_filter_windows.go

```diff
@@ -40,20 +41,22 @@ func (f *attrFilter) doesSupportThisOS() (msg string, supported bool) {
 
 // TODO: review handling of filename for long path support
 func (f *attrFilter) doesPass(storedObject storedObject) bool {
-	fileName := filepath.Join(f.filePath, storedObject.name)
+	fileName := filepath.Join(f.filePath, storedObject.relativePath)
```

Comment:
I thought there we cases where relativePath = "" (but this may be my on-going misunderstanding of that field ;-)  I must get I straight in my head sometime!)

Reply:
I might be able to clarify here. Actually `f.filePath` came from `sourceTraverser.fullPath`, which has whatever root folder the users put in the command line when they use `sync`...That's why I used `f.filePath`/`storedObject.name` as a full path. Did the test with `storedObject.relativePath` work?

Reply:
@panfeng0007 yes, the test works now. The storedObject.name only has the file name, for example, if the file is located in a sub-dir inside the full path(/users/xxxx/dir), "subdir1/filename.txt", then storedObject.name is just "filename.txt", while storedObject.relativePath is "subdir1/filename.txt".

If the full path pointed to a file already, e.g. /users/xxxx/dir/subdir1/filename.txt, then the storedObject.relativePath is empty "".

Reply:
Thanks @Ze.  I think I finally understand that field now too! :-)